.pre-commit-config.yaml

```
@@ -19,27 +19,25 @@ repos:
         args: ["--fix=crlf"]
         files: \.bat$
   - repo: https://github.com/shellcheck-py/shellcheck-py
-    rev: v0.8.0.4
+    rev: v0.9.0.6
     hooks:
       - id: shellcheck
   - repo: https://github.com/pre-commit/mirrors-prettier
-    rev: v2.7.1
+    rev: v3.1.0
     hooks:
       - id: prettier
-  - repo: https://github.com/pycqa/isort
-    rev: 5.12.0
+  - repo: https://github.com/astral-sh/ruff-pre-commit
+    rev: v0.3.0
     hooks:
-      - id: isort
-  - repo: https://github.com/psf/black
-    rev: 22.6.0
+      - id: ruff
+      - id: ruff-format
```

This is how I use it

```suggestion
      - id: ruff
        name: ruff lint
        args: [--fix]
      - id: ruff
        name: ruff sort imports
        args: ["--select", "I", "--fix"]
      - id: ruff-format
        name: ruff format
```

Maybe worth testing if there's a difference in end-results